opencti-platform/opencti-front/src/private/components/settings/users/edition/ConfidenceOverrideField.tsx

```diff
@@ -26,18 +26,21 @@ interface UserConfidenceOverridesFieldComponentProps
   index: number;
   onDelete: () => void;
   onSubmit: (index: number, value: OverrideFormData | null) => void;
+  currentOverrides: OverrideFormData[];
 }
 
-const UserConfidenceOverrideField: FunctionComponent<UserConfidenceOverridesFieldComponentProps> = ({
+const ConfidenceOverrideField: FunctionComponent<UserConfidenceOverridesFieldComponentProps> = ({
   form,
   field,
   index,
   onDelete,
   onSubmit,
+  currentOverrides,
 }) => {
   const { t_i18n } = useFormatter();
   const theme = useTheme<Theme>();
   const { availableEntityTypes } = useSchema();
+  const entityTypesToOverride = availableEntityTypes.filter((entity_type) => entity_type.type !== 'entity_Stix-Meta-Objects' && entity_type.type !== 'entity_Stix-Cyber-Observables');
```

Comment:
Badass cette ligne de code

Could be a function or at least separate into multiple lines?